### PR TITLE
Evolution: add 2.13, update example

### PR DESCRIPTION
**Pull request by kwalrath**
Staged:

https://kw-www-dartlang-1.firebaseapp.com/guides/language/evolution

### CODE REVIEW

File: src/_guides/language/evolution.md [hunk 1]
```diff
@@ -180,6 +180,18 @@ turn into edit-time analysis errors.
 
 In Dart 2.12, **[Dart FFI][]** graduated from beta to the stable channel.
 
+### Dart 2.13
+
+Dart 2.13 added support for **[type aliases][]**,
```
Comment (parlough) [inline src/_guides/language/evolution.md:185]: Do we need to specify that support was added for "non-function type aliases" or were the function typedefs not referred as type aliases originally?

**Review by parlough — APPROVED** [review]
Looks good to me, one question though.

File: src/_guides/language/evolution.md (hunk 1 above)
Comment (mit-mit) [reply src/_guides/language/evolution.md:185]: Yeah, either "added support for non-function type aliases" or perhaps "added complete support for type aliases"?

**Review by mit-mit — APPROVED** [review]

File: src/_guides/language/evolution.md (hunk 1 above)
Comment (kwalrath) [reply src/_guides/language/evolution.md]: I tried something slightly different:

Dart 2.13 expanded support for type aliases (typedef), which used to work only for function types but now work for any type. 

wdyt?

**Pull request merged**